packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -56,6 +59,8 @@ function TooltipTrigger(props: TooltipTriggerProps, ref: DOMRef<HTMLElement>) {
     placement: props.placement || 'right',
     targetRef: tooltipTriggerRef,
     overlayRef,
+    offset: props.offset || DEFAULT_OFFSET,
```

Comment:
We should probably add offset/crossOffset directly to TooltipTriggerProps so that we can annotate them with the default values
Would you mind doing that?

Reply:
You got it 👍

Reply:
@snowystinger - Done. Please have a 👀